app/js/components/manageApps/ManageApps.jsx

```diff
@@ -859,14 +859,14 @@ export default class ManageApps extends React.Component {
               <span className="pull-right manage-settings-wrapper">
                 <span id="startall-modules-btn"
                   className="btn btn-secondary"
-                  onClick={updatesAvailable? this.clearUpdates : this.checkForUpdates}>{updatesAvailable? 'Back to all Addons': 'Check For Updates'}</span>
+                  onClick={updatesAvailable? this.clearUpdates : this.checkForUpdates}>{updatesAvailable? 'Back to all Addons': 'Check for Updates'}</span>
```

Comment:
you forgot to add a space between the variable and the question mark

Reply:
@brayoh Thank you, I have updated the PR